src/phenomic-loader/__tests__/index.js

```diff
@@ -86,6 +86,17 @@ test.cb("phenomic loader", (t) => {
         "should create a proper json for custom route with an extension"
       )
 
+      const customRouteRootIndex = stats.compilation.assets[
+        //    fixtures/custom-route-root-index.md
+        // -> fixtures/index.html
+        "index.html" +
+        ".8817d2a1fab9dfb9b4b52cd6ee7529ab.json"
+      ]
+      t.truthy(
+        customRouteRootIndex && customRouteRootIndex._value,
+        "should create a proper json for custom route with an extension"
+      )
```

Comment:
This test does not seems to add some value here. What is this supposed to check?